collect_app/src/main/java/org/odk/collect/android/utilities/ReferenceLayerUtils.kt

```diff
@@ -0,0 +1,20 @@
+package org.odk.collect.android.utilities
+
+import android.os.Bundle
+import org.odk.collect.android.geo.MapFragment
+import org.odk.collect.shared.PathUtils.getAbsoluteFilePath
+import java.io.File
+
+object ReferenceLayerUtils {
+
+    @JvmStatic
+    fun getReferenceLayerFile(config: Bundle, layersPath: String?): File? {
```

Comment:
This method wouldn't make sense in `geo` as it feels like it is more "collect specific".